test/new-features.md

```diff
@@ -0,0 +1,138 @@
+# Lightning v1.0 Features
+
+Lightning is out of beta! With our official release, we'd like
+to highlight our past accomplishments and some new features.
+
+Core developers: 
+
+* Alex Fong
+* Frank Chen
+* Ky-Cuong Huynh
+
+
+## New Lightning Features
+
+### Markdown Rendering
+
+All Markdown files (like [this one](https://github.com/UCLA-CS130/Mr.-Robot-et-al./blob/e34b0fcb693a28f48e4270a6f0e38967f05a3b65/test/test.md))
+will now automatically be rendered into Markdown-style HTML before being 
+served up. Previously, such files would have been served as plain text, 
+which is off-putting to many users and difficult to read.
+
+This design document is served up in this way. It's a perfect example of
+how Markdown improves readability and the delightfulness of a Lightning-powered
+user experience.
+
+To render the Markdown, we use the excellent [Cpp-Markdown](https://github.com/KyCodeHuynh/Cpp-Markdown) library.
+
+
+#### Demo
+
+For a running Lightning server, visit any one of these paths: 
+
+    http://ec2-54-242-5-206.compute-1.amazonaws.com/static/test.md
```

Comment:
Changes mentioned [above](https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/77/files#r105546765) should be addressed in this file